### PR TITLE
Fix smiley viewer script so that it runs without errors

### PR DESCRIPTION
**Pull request by djmattyg007**

### CODE REVIEW

File: tools/collect_smileys.php [hunk 1]
```diff
@@ -67,7 +67,6 @@ function collect_smileys($file) {
 	table.smiley_table td { font: 10pt Courier,monospace,mono; white-space: pre; }
 	table.smiley_table tbody th { text-align: center; }	
 --></style>
-<base href="<?php print "http://" . $_SERVER['SERVER_NAME'] . dirname(dirname($_SERVER['REQUEST_URI'])); ?>" />
```
Comment (DaazKu) [inline tools/collect_smileys.php:70]: This line should not be removed. 

See https://github.com/vanilla/nbbc/pull/11/files#diff-028043b7bd7f006af37bbd7f9649a099R29

**Review by DaazKu — CHANGES_REQUESTED** [review]

File: tools/collect_smileys.php (hunk 1 above)
Comment (djmattyg007) [reply tools/collect_smileys.php:70]: Did you mean to link somewhere else? Right now you're linking to this diff.

Comment (DaazKu) [reply tools/collect_smileys.php:70]: Seems like github doesn't auto expand the code.
https://github.com/djmattyg007/nbbc/blob/820e64739b6d6923f160d0201384bd0663889e88/tools/collect_smileys.php#L29

TL;DR: that script is meant to be called from a server so that line should not raise any error.

**Comment by djmattyg007** [comment]
I removed the line because I erroneously assumed the `<base>` tag only affected `<a>` tags. I'll put it back in, thanks for raising it.

**Review by DaazKu — APPROVED** [review]
